app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.11
+
+WORKDIR /app
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+COPY . .
+
+CMD ["python", "app.py"]
```

Comment:
There is a 'proper' way to run flask using the [Flask CLI](https://flask.palletsprojects.com/en/2.2.x/cli/), which we could set as an entrypoint (and then pass in any args we want via docker-compose or manually running the container), but that's not strictly necessary.

At some point in the future we should rejig this (or have a separate production dockerfile) to use a WSGI server, see [the flask docs on deployment](https://flask.palletsprojects.com/en/2.2.x/deploying/)

Reply:
The idea here is to 100% have a separate production dockerfile, so I'm happy running in debug here.

Reply:
It also looks like you can do some other nice things like [custom commands](https://flask.palletsprojects.com/en/2.2.x/cli/#registering-commands-with-blueprints) which we might want to use in the future to make administering camvote easier

Reply:
That's probably going to be needed for "first time DB setup" (and migrations? but i'm gonna pretend we'll never need to do migrations)